packages/blade/src/utils/getFocusRingStyles/focusRingTypes.ts

```diff
@@ -0,0 +1,8 @@
+import type { Theme } from '~components/BladeProvider';
```

Comment:
just call this file `types.ts`

This is a common convention we are following in all components. `types.ts` containing common types for a component.